core/src/main/java/org/apache/gravitino/storage/relational/service/CatalogMetaService.java

```diff
@@ -215,6 +215,10 @@ public boolean deleteCatalog(NameIdentifier identifier, boolean cascade) {
               SessionUtils.doWithoutCommit(
                   FilesetMetaMapper.class,
                   mapper -> mapper.softDeleteFilesetMetasByCatalogId(catalogId)),
+          () ->
+              SessionUtils.doWithoutCommit(
```

Comment:
Seems incorrect here, should delete column meta but delete fileset meta.

Reply:
Sorry my fault, I will fix it.